Operator crashes when Spec.StartVM is not present in the CR

```
E0409 15:29:22.486943   33419 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 879 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x1ecee00, 0x347aa80)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:74 +0xa3
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:48 +0x82
panic(0x1ecee00, 0x347aa80)
        /usr/lib/golang/src/runtime/panic.go:679 +0x1b2
github.com/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport.(*ReconcileVirtualMachineImport).startVM(0xc000282e00, 0x24c72a0, 0xc0001fe000, 0xc0000c81a0, 0xc000475700, 0x7, 0x0, 0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go:307 +0x43
github.com/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport.(*ReconcileVirtualMachineImport).Reconcile(0xc000282e00, 0xc000475700, 0x7, 0xc0004756f0, 0x7, 0xc000ca5c00, 0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go:226 +0x95a
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc00076a240, 0x1f57c80, 0xc0008c5240, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:256 +0x162
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc00076a240, 0xc00061f000)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:232 +0xcb
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker(0xc00076a240)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:211 +0x2b
k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc000690000)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:152 +0x5e
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc000690000, 0x3b9aca00, 0x0, 0x1, 0xc00063f380)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:153 +0xf8
k8s.io/apimachinery/pkg/util/wait.Until(0xc000690000, 0x3b9aca00, 0xc00063f380)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88 +0x4d
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:193 +0x328
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1c05613]

goroutine 879 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0x105
panic(0x1ecee00, 0x347aa80)
        /usr/lib/golang/src/runtime/panic.go:679 +0x1b2
github.com/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport.(*ReconcileVirtualMachineImport).startVM(0xc000282e00, 0x24c72a0, 0xc0001fe000, 0xc0000c81a0, 0xc000475700, 0x7, 0x0, 0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go:307 +0x43
github.com/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport.(*ReconcileVirtualMachineImport).Reconcile(0xc000282e00, 0xc000475700, 0x7, 0xc0004756f0, 0x7, 0xc000ca5c00, 0x0, 0x0, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go:226 +0x95a
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc00076a240, 0x1f57c80, 0xc0008c5240, 0x0)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:256 +0x162
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc00076a240, 0xc00061f000)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:232 +0xcb
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker(0xc00076a240)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:211 +0x2b
k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc000690000)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:152 +0x5e
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc000690000, 0x3b9aca00, 0x0, 0x1, 0xc00063f380)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:153 +0xf8
k8s.io/apimachinery/pkg/util/wait.Until(0xc000690000, 0x3b9aca00, 0xc00063f380)
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88 +0x4d
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
        /home/jdzon/projects/kubevirt/vm-import-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:193 +0x328
```

--- a/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go
+++ b/pkg/controller/virtualmachineimport/virtualmachineimport_controller.go
@@ -193,7 +193,7 @@ func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (re
 				}
 
 				// Cleanup if user don't want to start the VM
-				if !*instance.Spec.StartVM {
+				if instance.Spec.StartVM == nil || !*instance.Spec.StartVM {
 					if err := r.updateProgress(instance, progressDone); err != nil {
 						return reconcile.Result{}, err
 					}
@@ -288,7 +288,7 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 
 // startVM start the VM if was requested to be started and VM disks are imported and ready:
 func (r *ReconcileVirtualMachineImport) startVM(provider provider.Provider, instance *v2vv1alpha1.VirtualMachineImport, vmName types.NamespacedName) error {
-	if *instance.Spec.StartVM && conditions.HasSucceededConditionOfReason(instance.Status.Conditions, v2vv1alpha1.VirtualMachineReady) {
+	if shouldStartVM(instance) {
 		vmi := &kubevirtv1.VirtualMachineInstance{}
 		err := r.client.Get(context.TODO(), vmName, vmi)
 		if err != nil && errors.IsNotFound(err) {
@@ -319,6 +319,10 @@ func (r *ReconcileVirtualMachineImport) startVM(provider provider.Provider, inst
 	}
 
 	return nil
+}
+
+func shouldStartVM(instance *v2vv1alpha1.VirtualMachineImport) bool {
+	return instance.Spec.StartVM != nil && *instance.Spec.StartVM && conditions.HasSucceededConditionOfReason(instance.Status.Conditions, v2vv1alpha1.VirtualMachineReady)
 }
 
 // manageDataVolumeState update current state according to progress of import

